btr-web/btr-common-components/components/bcros/DatePicker.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="bcros-date-picker" :class="{ 'bcros-date-picker__err': error }">
+    <VueDatePicker
+      v-model="selectedDate"
+      auto-apply
+      :action-row="{ showCancel: false, showNow: false, showPreview: false, showSelect: false }"
+      calendar-cell-class-name="bcros-date-picker__calendar__day"
+      calendar-class-name="bcros-date-picker__calendar"
+      :day-names="['SUN', 'MON','TUE','WED','THU','FRI','SAT']"
+      :enable-time-picker="false"
+      format="yyyy-MM-dd"
+      hide-offset-dates
+      inline
+      :max-date="maxDate || ''"
+      :min-date="minDate || ''"
+      :month-change-on-scroll="false"
+      :week-start="0"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Ref, ref, watch } from 'vue'
+import VueDatePicker from '@vuepic/vue-datepicker'
+import '@vuepic/vue-datepicker/dist/main.css'
+
+// props / emits
+const props = defineProps<{
+  defaultSelectedDate?: Date | null
+  error?: boolean
+  setMaxDate?: Date
+  setMinDate?: Date
+}>()
+const emit = defineEmits<{(e: 'selectedDate', value: Date | null): void }>()
+
+// date selection
+const selectedDate: Ref<Date | null> = ref(props.defaultSelectedDate || null)
+watch(() => selectedDate.value, val => emit('selectedDate', val))
+
+// max/min
+const maxDate: Ref<Date | null> = ref(props.setMaxDate || null)
+watch(() => props.setMaxDate, val => maxDate.value = val || null)
+
+const minDate: Ref<Date | null> = ref(props.setMinDate || null)
+watch(() => props.setMinDate, val => minDate.value = val || null)
+
+</script>
+<style lang="scss">
+.bcros-date-picker {
+  width: 300px;
+
+  &__err {
+    border: 1px solid $error;
+    border-radius: 10px;
+    box-shadow: 0px 0px 3px $error;
+  }
+
+  &__calendar {
+
+    &__day {
+      border-radius: 50%;
+    }
+
+    .dp__calendar_header {
+      margin-top: 6px;
+
+      .dp__calendar_header_item {
+        font-size: 12px;
+        font-weight: 500;
+        color: $gray9 !important;
+        padding: 4px 0 0 0;
+        width: 40px;
+      }
+    }
+  }
+
+  .dp__menu {
+    border-radius: 0 0 10px 10px;
+    height: 329px;
+  }
+
+  .dp__main {
+    width: 298px;
+  }
+
+  .dp__theme_light {
+    --dp-background-color: #ffffff;
```

Comment:
bah actually I can't pass those in for the .dp__theme__light override for some reason

Reply:
We can see if its viable in the future, I was just thinking if it would be easy to do so it would be good.
Anyway it has its own theme so I would say it does not matter much for now.